src/configs/endpoints.js

```diff
@@ -16,7 +16,7 @@ export const authEndpoints = {
   signupCompany: `${baseEndpoints.auth}organizations/signup`,
   signinCompany: `${baseEndpoints.auth}organizations/login`,
   activate: `${baseEndpoints.auth}/register/activate`,
-  resetPassword: `${baseEndpoints.auth}password`,

```

Comment:
this resetpassword endpoint is already being used elsewhere, before you change anything, first confirm if it is used elsewhere

Reply:
Okay